srcpkgs/hplip/template

```diff
@@ -31,12 +32,12 @@ makedepends="openssl-devel python3-devel libxml2-python3 cups-devel sane-devel
 depends="python3-gobject python3-dbus desktop-file-utils
  foomatic-db foomatic-db-engine python3-distro"
 short_desc="HP Linux Imaging and Printing"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="dataCobra <datacobra@thinkbot.de>"
```

Comment:
Don't adopt yet. We want to see a longer contribution history than one month, and at least a few cycles of updates to this package, before putting somebody's name on it.

Reply:
This is new to me... I'm already the maintainer for mercurial for which I've only created two PRs.

But I'll undo this change then.